README.md

```diff
@@ -7,18 +7,20 @@ The build information is sent to Artifactory in json format.
 
 ## Building and Testing the Sources
 
-The code is built using Gradle and includes integration tests.<br/>
-It must run using JDK 8 and Gradle 5.6.2. If you are using different gradle version you can use the provided gradle wrapper.<br/>
-In order to run tests the following environment variable must be provide:
+* The code is built using Gradle and includes integration tests.<br/>
+* It must run using JDK 8 and Gradle 5.6.2. If you are using different gradle version you can use the provided gradle wrapper.<br/>
+* In order to run tests the following environment variable must be provided:
 ```
 export BITESTS_ARTIFACTORY_URL='http://localhost:8081/artifactory'
 export BITESTS_ARTIFACTORY_USERNAME=admin
 export BITESTS_ARTIFACTORY_PASSWORD=password
 export BITESTS_ARTIFACTORY_REPO=tests
 export BITESTS_ARTIFACTORY_PIP_ENV=/Users/user/venv-test/bin
 ```
-Before running the tests, please make sure you have a generic repository named *tests* in Artifactory.
+* Before running the tests, please make sure you have a generic repository named *tests* in Artifactory.
+* Please follow the testing instructions below for more details of different types of build-tools test instructions.
```

Comment:
Please follow the testing instructions below for more details of different types of build-tools test instructions.

I suggest removing the above sentence, and adding the following one instead, but placing it as part of the *Building* section, because it is related specifically to the ```gradle clean build``` command:

When running the following command to build the code, the entire testing suite is also executed. See the *Testing* section for configuration instructions prior to runni g the tests.

Reply:
Added.